.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+Thank you for your pull request!  
+
+Please add '[major]', '[minor]', or [patch] to the title to indicate the impact the change has on the code.  Please also provide the following information.
+
+---
+
+1. **Do these changes impact *command syntax* of any of the packages?** (e.g., add/remove command, add/remove a command parameter, or update required parameters)
+    If Yes, briefly describe what is impacted.
+
+
+2. **Do these changes impact *documentation*?** (e.g., a tutorial on https://docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins)
+    If Yes, briefly describe what is impacted.
+
+
+If you answered yes to any of these please do the following:
+    > Include 'Rick-Kirkham' in the review
+    > Make sure the README file is correct
+
+Change description:
```

Comment:
What is this line about? I found confusing what this means.
Maybe just the `:` is giving me confusion.
Also, as the other PR, would be nice to add a question about how they tested their feature

Reply:
It just suggesting further change description in the lines below.